graphein/ppi/features/node_features.py

```diff
@@ -27,7 +28,7 @@ def add_sequence_to_nodes(n, d):
     # Todo mapping with bioservices to support other protein IDs?
 
     for id in d["uniprot_ids"]:
-        d[f"sequence_{id}"] = u.get_fasta_sequence(id)
+        d[f"sequence"] = u.get_fasta_sequence(id)
```

Comment:
No need for f-string anymore, since no variable is being inserted?

```suggestion
        d["sequence"] = u.get_fasta_sequence(id)
```